warehouse/query-core/src/main/java/datawave/query/common/grouping/GroupingUtil.java

```diff
@@ -0,0 +1,316 @@
+package datawave.query.common.grouping;
+
+import com.google.common.collect.*;
```

Comment:
No star imports, please

Reply:
Fixed and changed the setting in intellij so it won't happen again.